app/code/Magento/Checkout/view/frontend/web/js/product-prices-updated.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+
+define([
+    'jquery',
+    'Magento_Customer/js/customer-data'], function ($, customerData) {
+    'use strict';
+
+    return function () {
+        var cacheKey = 'cart-cache-updated',
+            cartData = customerData.get('cart'),
+            localStorage = $.initNamespaceStorage('mage-cache-storage').localStorage,
```

Comment:
Please don't use this shitty library, it's too buggy

Please use native localstorage